CHANGES.md

```diff
@@ -15562,7 +15562,7 @@ s-cbc           3624.96k     5258.21k     5530.91k     5624.30k     5628.26k
 
    *Tim Mooney <mooney@dogbert.cc.ndsu.NoDak.edu> via Richard Levitte*
 
- * In ssl3_get_key_exchange (ssl/s3_clnt.c), call ssl3_get_message()
+ * In ssl3_get_key_exchange (ssl/s3_clnt.c), call tls_get_message_body()
    with the same message size as in ssl3_get_certificate_request().
```

Comment:
Do not edit the CHANGES.md as that should reflect the history as of when the concrete change was done and not the current state of the code.

Reply:
Okay

Reply:
You still did not revert this change.

Reply:
I am sorry I forgot this one.